.github/actions/docker/action.yml

```diff
@@ -63,13 +63,9 @@ inputs:
     default: 'false'
     description: 'Generate tag only.'
     required: false
-  CACHE_FROM:
```

Comment:
these are not used by any consuming repos and not exposed in the ci workflows so removing them